scripts/install/etc/sysconfig/motr

```diff
@@ -151,3 +151,6 @@ MOTR_TRACED_KEEP_LOGS_NUM=2
 # for developers to quickly test their local changes w/o creating and installing
 # an rpm package each time.
 #MOTR_DEVEL_WORKDIR_PATH=/home/devvm/motr
+
+#MOTR ADDB Record default record size in bytes (1G)
+MOTR_MOD_ADDB_RECORD_SIZE=134217728
```

Comment:
This is 128M, not 1G.

Reply:
@huanghua78 : It means if we don't set here default is 1G.

Reply:
@huanghua78 : You are right, default is 128M.